new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/PersistentVolumeClaim.java

```diff
@@ -7,14 +7,17 @@
 import io.kubernetes.client.openapi.models.V1PersistentVolumeClaim;
 import oracle.weblogic.kubernetes.actions.impl.primitive.Kubernetes;
 
+/**
+ * TestAction class PersistentVolumeClaim to create and delete a Kubernetes Persistent Volume.
+ *
+ */
 public class PersistentVolumeClaim {
 
   /**
    * Create a Kubernetes Persistent Volume Claim.
    *
-   * @param persistentVolumeClaim V1PersistentVolumeClaim object containing Kubernetes
-   *     persistent volume claim configuration data
-   * @return true if successful
+   * @param persistentVolumeClaim object containing persistent volume claim configuration data
+   * @return true if successful false otherwise
    * @throws ApiException if Kubernetes client API call fails
    */
   public static boolean create(V1PersistentVolumeClaim persistentVolumeClaim) throws ApiException {
```

Comment:
Strange... you are using the standard types here.  I don't understand then why you created your own types above.

Reply:
The reason I created my own type is avoid crowding the testMethod with so much lines of code dealing with same stuff in every IT class. The new types could provide less lines of code and need to set only the configuration needed by PersistentVolume and PersistentVolumeClaim on the POJO objects.

Removed the newly created classes and added everything into the testMethod.